migrations/Version20210612055018.php

```diff
@@ -20,6 +20,7 @@ public function getDescription(): string
     public function up(Schema $schema): void
     {
         // this up() migration is auto-generated, please modify it to your needs
+        // this is for what?
```

Comment:
for Symfony messenger async. 
https://symfony.com/doc/current/messenger.html#transports-async-queued-messages